libraries/core/lib/util/Datapoint.ts

```diff
@@ -16,24 +16,28 @@
  * limitations under the License.
  */
 
+import { EdgeType } from "@syntest/cfg-core";
+
 /**
  * @author Dimitri Stallenberg
  */
 export interface Datapoint {
   id: string;
   type: string;
+
   path: string;
   line: number;
 
   hits: number;
 
-  locationIdx?: number;
-  branchType?: boolean;
-
   condition_ast?: string;
   condition?: string;
   variables?: unknown;
 
+  // deprecated
+  locationIdx?: number;
+  branchType?: boolean;
+
```

Comment:
I assume this is deprecated because we now use the ids to target the nodes instead of the edge with certain labels. However, would this also work with Solidity, where we have less control over the instrumentation and AST?

Reply:
Removed it entirely. In Solidity we can also create our own instrumentation